.ci/azure-pipelines/main-pr.yml

```diff
@@ -59,7 +59,7 @@ jobs:
     workspace:
       clean: all
     pool:
-      vmImage: 'macOS-latest'
+      vmImage: 'macOS-11'
```

Comment:
Why not leave `macOS-latest`?

Reply:
Because using `latest` on CI just translates to `Will unexpectedly fail at some point down the line because the env got updated`. We don't want that